vms/platformvm/blocks/executor/acceptor_test.go

```diff
@@ -19,13 +18,14 @@ import (
 	"github.com/ava-labs/avalanchego/utils"
 	"github.com/ava-labs/avalanchego/utils/logging"
 	"github.com/ava-labs/avalanchego/utils/timer/mockable"
-	"github.com/ava-labs/avalanchego/utils/window"
 	"github.com/ava-labs/avalanchego/vms/components/verify"
 	"github.com/ava-labs/avalanchego/vms/platformvm/blocks"
 	"github.com/ava-labs/avalanchego/vms/platformvm/metrics"
 	"github.com/ava-labs/avalanchego/vms/platformvm/state"
 	"github.com/ava-labs/avalanchego/vms/platformvm/txs"
 	"github.com/ava-labs/avalanchego/vms/secp256k1fx"
+
+	pvalidators "github.com/ava-labs/avalanchego/vms/platformvm/validators"
```

Comment:
We don't need to alias this import.
```suggestion
	"github.com/ava-labs/avalanchego/vms/platformvm/validators"
```

Reply:
cleaned up